pkg/auth/requests/authenticate.go

```diff
@@ -80,7 +86,7 @@ type tokenAuthenticator struct {
 	userAttributeLister v3.UserAttributeLister
 	userLister          v3.UserLister
 	clusterRouter       ClusterRouter
-	userAuthRefresher   providerrefresh.UserAuthRefresher
+	refreshUser         userRefresher
```

Comment:
This is to be able to deterministically test that user refresh (not) triggered. There is no good way to test for NOT running something in a fire-and-forget manner.